src/pages/StudentGroup/index - Copy.jsx

```diff
@@ -0,0 +1,169 @@
+      <Sidebar />
+      <div className="flex flex-col w-full">
+        <Header />
+        <div className="w-full px-64 pt-10">
+          <h1 className="mb-4 text-xl font-semibold text-gray-700 dark:text-gray-200">
+            Create Your Submission Here
+          </h1>
+          <label className="label">
+            <span className="label-text">Submission Type</span>
+          </label>
+          <input
+            className="mt-1 input input-bordered w-full"
+            type="text"
+            name="type"
+            placeholder="Enter submission type..."
+            value={inputs.type}
+            onChange={handleChange}
+            required
+          />
+
+          <label className="mt-4 label">
+            <span className="label-text">Submission Description</span>
+          </label>
+          <textarea
+            className="mt-1 input input-bordered w-full h-24"
+            type="text"
+            name="description"
+            placeholder="Enter Submission Description..."
+            value={inputs.description}
+            onChange={handleChange}
+            required
+          />
+
+          <button
+            className="mt-4 btn btn-active btn-primary w-full"
+            onClick={Submission}>
+            SAVE
+          </button>
+          <br />
+          <br />
+          <br />
+          <br />
+
+          <div class="relative overflow-x-auto shadow-md sm:rounded-lg">
+            <table class="w-full text-sm text-left text-gray-500 dark:text-gray-400">
+              <thead class="text-xs text-gray-700 uppercase bg-gray-50 dark:bg-gray-700 dark:text-gray-400">
+                <tr>
+                  <th scope="col" class="px-6 py-3">
+                    Type
+                  </th>
+                  <th scope="col" class="px-6 py-3">
+                    Description
+                  </th>
+                  <th scope="col" class="px-6 py-3"></th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                  <th
+                    scope="row"
+                    class="px-6 py-4 font-medium text-gray-900 dark:text-white whitespace-nowrap">
+                    <td class="px-6 py-1">
+                      {submission.map((sub) => (
+                        <div key={sub.id}>
+                          {sub.type}
+                          <br />
+                          <br />
+                        </div>
+                      ))}
+                    </td>
+                  </th>
+                  <td class="px-6 py-1">
+                    {submission.map((sub) => (
+                      <div key={sub.id}>
+                        {sub.description} <br />
+                        <br />
+                      </div>
+                    ))}
+                  </td>
+                  <td class="px-6 py-1">
+                    {submission.map((sub) => (
```

Comment:
Mapping should not contain submission (please change it to student group instead)